packages/ra-ui-materialui/src/layout/MenuItemLink.tsx

```diff
@@ -113,20 +110,28 @@ export const MenuItemLink = forwardRef((props: MenuItemLinkProps, ref) => {
             >
                 {leftIcon && (
                     <ListItemIcon className={MenuItemLinkClasses.icon}>
-                        {cloneElement(leftIcon, {
-                            titleAccess: primaryText,
```

Comment:
this is bad for accessibility, isn't it?

Reply:
Right, we should add a visually hidden text next to it

Reply:
Acutally it's not as we already have the text just after. It's only visually hidden when the sidebar is closed but still accessible by assistive technologies